stubs/AMQPDecimal.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+/**
+ * stub class representing AMQPDecimal from pecl-amqp
+ */
+final class AMQPDecimal
+{
+    const E_MIN = 0;
+    const E_MAX = 255;
+    const N_MIN = 0;
```

Comment:
Wanna call this significand or mantissa and therefore SIGNIFICAND_MAX/MIN or MANTISSA_MAX/MIN?